### PR TITLE
Singlepage

### PR DESCRIPTION
**Pull request by pariyar07**
Single page video and playlist feature added

### CODE REVIEW

File: src/reducer/videoReducer.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+                ...state, playlist: [...state.playlist, {...action.payload}]
+            }
+        }
+        case 'HISTORY':{
+            return {
+                ...state, history: [...state.history, {...action.payload}]
+            }
+        }
+        default:
+            return state;
```
Comment (ranitasahaa1230) [inline src/reducer/videoReducer.jsx:26]: Instead of returning state , you can also throw an error

**Review by ranitasahaa1230 — APPROVED** [review]
Code can be understood easily
Functionality is working fine.
Looks good to go

File: src/pages/homepage/styles.css [hunk 2]
```diff
@@ -0,0 +1,354 @@
+/* 
+home-nav
+*/
+.video-nav-content{
+    display: flex;
+    flex-direction: row;
```
Comment (Rockyraven) [inline src/pages/homepage/styles.css:6]: this comment line is good for easy to understand the code👌

**Review by Rockyraven — APPROVED** [review]
your code is clean, file management is also good
the way of using comment line is appreciable to easy to understand the code 
You can merge

File: src/reducer/videoReducer.jsx (hunk 1 above)
Comment (pariyar07) [reply src/reducer/videoReducer.jsx:26]: Will look into that.

**Pull request merged**